packages/gatsby-recipes/src/providers/npm/package-json.js

```diff
@@ -20,7 +20,7 @@ const writePackageJson = async (root, obj) => {
 }
 
 const create = async ({ root }, { name, value }) => {
-  const release = await lock(`package.json`)
+  const release = await lock(root + `package.json`)
```

Comment:
Are we sure root ends with a /? `path.join` gives us a bit more certainty.
```suggestion
  const release = await lock(path.join(root, `package.json`))
```